orcid-core/src/main/java/org/orcid/core/manager/v3/read_only/impl/SpamManagerReadOnlyImpl.java

```diff
@@ -0,0 +1,38 @@
+package org.orcid.core.manager.v3.read_only.impl;
+
+import javax.annotation.Resource;
+
+import org.orcid.core.adapter.v3.JpaJaxbSpamAdapter;
+import org.orcid.core.manager.v3.ProfileEntityManager;
+import org.orcid.core.manager.v3.read_only.SpamManagerReadOnly;
+import org.orcid.jaxb.model.v3.release.record.Spam;
+import org.orcid.persistence.dao.SpamDao;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SpamManagerReadOnlyImpl implements SpamManagerReadOnly {
+    
+    private static final Logger LOGGER = LoggerFactory.getLogger(SpamManagerReadOnlyImpl.class);
+    
+    @Resource
+    protected JpaJaxbSpamAdapter jpaJaxbSpamAdapter;
+    
+    @Resource
+    protected SpamDao spamDao;        
```

Comment:
Hi @DanielPalafox, 

This is the read only manager, so, you should have a SpamDao that should read from the replica databases.

Please let me know if you need some advice on how to do this.